tests/sles4sap/hana_cluster.pm

```diff
@@ -59,14 +59,17 @@ sub run {
         my $cluster_conf = get_var('USE_SAP_HANA_SR_ANGI') ? 'angi_hana_cluster.conf' : "hana_cluster_$sles4sap::resource_alias.conf";
         assert_script_run 'curl -f -v ' . autoinst_url . "/data/sles4sap/$cluster_conf -o /tmp/$cluster_conf";
         $cluster_conf = '/tmp/' . $cluster_conf;
+        my $nic = script_output("ip -o link show | awk -F ':' '{print \$2}' | grep -v 'lo'");
```

Comment:
Better use `network_tools::iface()`. See: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22033

I missed this as the `eth0` is not hard coded in code, but in templates.